README.md

```diff
@@ -104,18 +110,70 @@ new ProjenStruct(project, { name: 'MyProjectOptions'})
   .withoutDeprecated();
 ```
 
+### Exposing CDK Properties
+
+A common use-case for this project is to expose arbitrary overrides in CDK constructs. For example, you may want to
+provide common AWS Lambda configuration, but allow a consuming user to override any arbitrary property.
+
+To accomplish this, first create the new struct in your `.projenrc.js` file.
+
+```js
+const { ProjenStruct, Struct } = require('@mrgrain/jsii-struct-builder');
+const { awscdk } = require('projen');
+
+const project = new awscdk.AwsCdkConstructLibrary({
+  // your config - see https://projen.io/awscdk-construct.html
+});
+
+new ProjenStruct(project, { name: 'MyFunctionProps' })
+  .mixin(Struct.fromFqn('aws-cdk-lib.aws_lambda.FunctionProps'))
+  .withoutDeprecated()
+  .allOptional()
+  .omit('code'); // our construct always provides the code
+```
+
+Then, expose the new struct in your CDK construct.
```

Comment:
```suggestion
### AWS CDK properties

A common use-case of this project is to expose arbitrary overrides in CDK constructs.
For example, you may want to provide common AWS Lambda configuration, but allow a consuming user to override any arbitrary property.

To accomplish this, first create the new struct in your `.projenrc.ts` file.

```ts
import { ProjenStruct, Struct } from '@mrgrain/jsii-struct-builder';
import { awscdk } from 'projen';

declare project: awscdk.AwsCdkConstructLibrary;

new ProjenStruct(project, { name: 'MyFunctionProps' })
  .mixin(Struct.fromFqn('aws-cdk-lib.aws_lambda.FunctionProps'))
  .withoutDeprecated()
  .allOptional()
  .omit('code'); // our construct always provides the code
```

Then, expose the new struct in your CDK construct:
```